.travis.yml

```
@@ -159,20 +208,22 @@ script:
   # - export PYTHONPATH="$PYTHONPATH:./ci/"
 
   # ray tune tests
-  - ./ci/suppress_output python python/ray/tune/tests/test_dependency.py
+  - if [ $RAY_CI_TUNE_AFFECTED == "1" ]; then ./ci/suppress_output python python/ray/tune/tests/test_dependency.py; fi
   # `cluster_tests.py` runs on Jenkins, not Travis.
-  - python -m pytest --durations=10 --ignore=python/ray/tune/tests/test_cluster.py python/ray/tune/tests
+  - if [ $RAY_CI_TUNE_AFFECTED == "1" ]; then python -m pytest --durations=10 --ignore=python/ray/tune/tests/test_cluster.py python/ray/tune/tests; fi
 
   # ray rllib tests
-  - python/ray/rllib/tests/run_silent.sh tests/test_catalog.py
-  - python/ray/rllib/tests/run_silent.sh tests/test_filters.py
-  - python/ray/rllib/tests/run_silent.sh tests/test_optimizers.py
-  - python/ray/rllib/tests/run_silent.sh tests/test_evaluators.py
+  - if [ $RAY_CI_RLLIB_AFFECTED == "1" ]; then python/ray/rllib/tests/run_silent.sh python python/ray/rllib/tests/test_catalog.py; fi
```

```suggestion
  - if [ $RAY_CI_RLLIB_AFFECTED == "1" ]; then python/ray/rllib/tests/run_silent.sh tests/test_catalog.py; fi
```

This is causing test failures. I think it was the result of an incorrect rebase. Please make sure to look through the tests and see if they are passing.

---

What specific difference does the suggested change make?